### PR TITLE
Add aioresponses

### PR DESCRIPTION
**Pull request by MementoRC**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MementoRC** [comment]
@conda-forge/help-python, Hello, ready for review

File: recipes/aioresponses/fix-version.patch
Comment (BastianZim) [inline recipes/aioresponses/fix-version.patch:1]: What is the reason that this is not included upstream?

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/aioresponses/fix-version.patch
Comment (MementoRC) [reply recipes/aioresponses/fix-version.patch:1]: I have removed it, it's not necessary

**Comment by MementoRC** [comment]
Oh, looks like someone else got the package on conda-forge. SHould I simply close this?

**Comment by BastianZim** [comment]
Yes, you can just close it, if it's the same.

**Comment by MementoRC** [comment]
This is very puzzling, I created and worked on these proposals since Aug 23rd, yet the maintainer of the other one submitted his 2 days ago!
Just another ant being squashed?  I have to say my enthusiasm for contributing to conda-forge has been beaten into submission and scalded self-esteem

**Comment by BastianZim** [comment]
Sorry to hear that. As we are all volunteers and reviewing in our spare time, it can be that some recipes are missed. If you think that your recipe is not being noticed, please feel free to ping the review group again or post a comment in our chat room and someone should come by.

**Pull request closed without merging**